pkg/network/http/monitor_test.go

```diff
@@ -524,13 +564,25 @@ func requestNotIncluded(t *testing.T, allStats map[Key]*RequestStats, req *netht
 	}
 }
 
-func isRequestIncluded(allStats map[Key]*RequestStats, req *nethttp.Request) bool {
+func isRequestIncluded(allStats map[Key]*RequestStats, req *nethttp.Request) (bool, error) {
```

Comment:
Consider changing the name to isSingleRequestIncluded

Reply:
renamed